db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/task/helper/CronExpressionStyle.java

```diff
@@ -0,0 +1,11 @@
+package com.github.kagkarlsson.scheduler.task.helper;
+
+public enum CronExpressionStyle {
```

Comment:
What are your thoughts on shortening the name to `CronStyle`?

Reply:
Done